diff --git a/node_modules/jstransformer-remark/index.js b/node_modules/jstransformer-remark/index.js
index 23c2769..c9966ff 100644
--- a/node_modules/jstransformer-remark/index.js
+++ b/node_modules/jstransformer-remark/index.js
@@ -12,7 +12,7 @@ function prepareRemark(options, locals) {
   const processor = remark()
   const opts = extend({}, options, locals)
   const plugins = (opts.plugins && Array.isArray(opts.plugins)) ? opts.plugins : []
-  plugins.push(html)
+  // plugins.push(html)
   plugins.forEach(plugin => {
     processor.use(plugin)
   })
